### PR TITLE
created tox shorthand for building and uploading packages to pypi

### PR DESCRIPTION
**Pull request by leifwalsh**

### CODE REVIEW

**Comment by leifwalsh** [comment]
@thejunglejane updated the maintainer's guide

File: docs/maintaining.rst [hunk 1]
```diff
@@ -196,13 +196,9 @@ Uploading to PyPI
 Once you've tagged the latest version of marbles, pull from GitHub to
 make sure your clone is up to date and clean, build both ``sdist`` and
 ``wheel`` packages for all three packages, and upload them with
-`twine`_::
+`twine`_. We have a `tox`_ rule to automate building and uploading::
```
Comment (thejunglejane) [inline docs/maintaining.rst]: If we just run `tox` by itself, will it try to do the pypi upload too?

Comment (leifwalsh) [reply docs/maintaining.rst]: Nope, it's not listed in `[tox]envlist`, which is the default set of envs to build with.

Comment (thejunglejane) [reply docs/maintaining.rst]: Great, ty!

**Pull request merged**